lib/athena_web/router.ex

```diff
@@ -53,7 +53,10 @@ defmodule AthenaWeb.Router do
     resources "/locations", LocationController, except: @subresource_actions
     resources "/events/:event/item_groups", ItemGroupController, only: @subresource_actions
     resources "/item_groups", ItemGroupController, except: @subresource_actions
-    resources "/item_groups/:item_group/items", ItemController, only: @subresource_actions
+
+    resources "/item_groups/:item_group/items", ItemController,
+      only: @subresource_actions -- [:index]
```

Comment:
@screeper Maybe sobelow is also complaining about this. In that case, write out the list.